### PR TITLE
Add naming suffix wrapper

### PR DESCRIPTION
**Pull request by MichalPloski**
Create custom python script that add suffix to monitoring instance to distunguish instances in ha mode

### CODE REVIEW

File: docker/k8s/README.md [hunk 1]
```diff
@@ -18,3 +18,13 @@ docker run --net=host  -e PGBOUNCER_NAME="test-namespace" -e NEWRELIC_KEY="dd" -
...
 ```
+
+5. Why we need get_k8s_pod_suffix.py file
+
+newrelic-plugin-agent uses newRelic plugin api which doesn't allow to distinquish more than 1 instance of the same plugin name.
+So if we have pgbouncer in a high-availability mode we can't see detailed monitoring divided by instance.
+To overcome this limitation we use script that connect to k8s api and check deployed pgbouncer pods. 
+If there is more than one pod script sorts them in descending order and return pod index in the list. 
+This suffix is added to PGBOUNCER_NAME so those two instances will be visible as two separated pgbouncer deployment in monitoring
+We need to ensure that rolling update policy is changed to .
```
Comment (zichul) [inline docker/k8s/README.md]: changed to what?

File: docker/k8s/bin/test_get_k8s_pod_suffix.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        if isinstance(self.rs, list):
+            for replicaset in self.rs:
+                if replicaset['metadata']['name'] == name:
+                    return DictAttributes(replicaset)
+        return None
+
+
+# Tests
+@mock.patch('get_k8s_pod_suffix.k8s_client_config')
+def test_suffix_is_not_set_for_one_pod(mocked_config):
```
Comment (medzin) [inline docker/k8s/bin/test_get_k8s_pod_suffix.py:10]: Put the tests on top of the file and other code below - for readability.

**Review by medzin — CHANGES_REQUESTED** [review]

File: docker/k8s/README.md (hunk 1 above)
Comment (MichalPloski) [reply docker/k8s/README.md]: fixed

File: docker/k8s/bin/test_get_k8s_pod_suffix.py (hunk 2 above)
Comment (MichalPloski) [reply docker/k8s/bin/test_get_k8s_pod_suffix.py:10]: Moved test data to separate file and moved helper classes to bottom of this file

File: docker/k8s/bin/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+kubernetes
```
Comment (medzin) [inline docker/k8s/bin/requirements.txt]: Lock versions.

**Review by medzin — CHANGES_REQUESTED** [review]

**Review by zichul — COMMENTED** [review]
tbh this looks like overkill just to get pod number but I guess we need to live with that

**Comment by MichalPloski** [comment]
> tbh this looks like overkill just to get pod number but I guess we need to live with that

It is how kubernetes works and I couldn't find easier way to do this and during development few cases showed up that needed to be covered.

File: docker/k8s/bin/requirements.txt (hunk 3 above)
Comment (MichalPloski) [reply docker/k8s/bin/requirements.txt]: locked

**Review by medzin — APPROVED** [review]
lgtm

**Pull request merged**